packages/@react-aria/focus/docs/FocusRing.mdx

```diff
@@ -59,15 +60,51 @@ This example shows how to use `<FocusRing>` to apply a CSS class when keyboard f
   font-size: 14px;
   padding: 4px 8px;
 }
-
+.button:disabled {
+background: #6f6f6f;
+}
 .button.focus-ring {
   outline: 2px solid dodgerblue;
   outline-offset: 2px;
 }
+
+.button + .button {
+  margin-inline-start: 6px;
+}
 ```
 
 ```tsx example
 <FocusRing focusRingClass="focus-ring">
   <button className="button">Test</button>
 </FocusRing>
 ```
+
+### Disabled
```

Comment:
🤔 this seems like a very specific example to put in the docs...

Reply:
yeah, i had that thought too, but it's come up a number of times, maybe blog would be a better place for it?